Library/Formula/llvm.rb

```diff
@@ -30,11 +30,21 @@ class Llvm < Formula
       sha256 "ae9180466a23acb426d12444d866b266ff2289b266064d362462e44f8d4699f3"
     end
 
+    resource "extra-tools" do
```

Comment:
IIRC I moved it so the resources are in alphabetical order. Do you still want it under the `lldb` resource?